roles/openstack-stack/templates/heat_stack_server.yaml.j2

```diff
@@ -83,6 +89,11 @@ parameters:
       - range: { min: 1, max: 1024 }
         description: must be between 1 and 1024 Gb.
 
+  node_labels:
+    type: json
+    description: OpenShift Node Labels
+    default: {"region": "default" }
```

Comment:
We apply `region:infra` to the infra nodes, but heat requires a default value for the parameter. I didn't want to make it something that actually has meaning.